Bottom: a8df00e32eb302f25f461a1ad37449e27699a5c5
Top:    3d59313a044050cfa62d6b4447dfb35976e684a4
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-08 10:57:35 +0800

Refresh of debug-2.patch

---

diff --git a/lib/master.c b/lib/master.c
index e3dc2cf..47c5ac9 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -899,8 +899,6 @@ int master_notify_submount(struct autofs_point *ap, const char *path, enum state
 
 	mounts_mutex_lock(ap);
 
-	error(LOGOPT_ANY, "ap->path %s", ap->path);
-
 	head = &ap->submounts;
 	p = head->prev;
 	while (p != head) {
@@ -909,7 +907,6 @@ int master_notify_submount(struct autofs_point *ap, const char *path, enum state
 
 		if (!master_submount_list_empty(this)) {
 			mounts_mutex_unlock(ap);
-			error(LOGOPT_ANY, "notify submount %s", this->path);
 			if (!master_notify_submount(this, path, state)) {
 				ret = 0;
 				mounts_mutex_lock(ap);
@@ -967,10 +964,7 @@ int master_notify_submount(struct autofs_point *ap, const char *path, enum state
 			st_mutex_lock();
 		}
 		st_mutex_unlock();
-		mounts_mutex_unlock(ap);
-
-		return ret;
-
+		break;
 	}
 
 	mounts_mutex_unlock(ap);
